src/azure-cli/azure/cli/command_modules/maps/_help.py

```diff
@@ -18,29 +18,48 @@
 """
 
 helps['maps account create'] = """
-type: command
-short-summary: Create a maps account.
-parameters:
-  - name: --accept-tos
-    short-summary: Accept the Terms of Service, and do not prompt for confirmation.
-    long-summary: |
-        By creating an Azure Maps account, you agree that you have read and agree to the
-        License (https://azure.microsoft.com/support/legal/) and
-        Privacy Statement (https://privacy.microsoft.com/privacystatement).
+    type: command
+    short-summary: "Create a Maps Account. A Maps Account holds the keys which allow access to the Maps REST APIs."
+    parameters:
+      - name: --linked-resources
+        short-summary: "Sets the resources to be used for Managed Identities based operations for the Map account \
+resource."
+        long-summary: |
+            Usage: --linked-resources unique-name=XX id=XX
 
-examples:
-  - name: Create a maps account. (autogenerated)
-    text: az maps account create --name MyMapsAccount --resource-group MyResourceGroup --sku S0 --subscription MySubscription
-    crafted: true
+            unique-name: Required. A provided name which uniquely identifies the linked resource.
+            id: Required. ARM resource id in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupNa\
+me}/providers/Microsoft.Storage/accounts/{storageName}'.
+
+            Multiple actions can be specified by using more than one --linked-resources argument.
+    examples:
+      - name: Create Account with Managed Identities
+        text: |-
+               az maps account create --type "SystemAssigned, UserAssigned" --user-identities \
+"{\\"/subscriptions/21a9967a-e8a9-4656-a70b-96ff1c4d05a0/resourceGroups/myResourceGroup/providers/Microsoft.ManagedIden\
+tity/userAssignedIdentities/identityName\\":{}}" --kind "Gen2" --disable-local-auth false \
+--linked-resources id="/subscriptions/21a9967a-e8a9-4656-a70b-96ff1c4d05a0/resourceGroups/myResourceGroup/providers/Mic\
+rosoft.Storage/accounts/mystorageacc" unique-name="myBatchStorageAccount" --linked-resources \
+id="/subscriptions/21a9967a-e8a9-4656-a70b-96ff1c4d05a0/resourceGroups/myResourceGroup/providers/Microsoft.Storage/acco\
+unts/mystorageacc" unique-name="myBlobDataSource" --name "G2" --tags test="true" --account-name "myMapsAccount" \
+--resource-group "myResourceGroup" --sku "S0"
+      - name: Create Gen1 Account
+        text: |-
+               az maps account create --kind "Gen1" --disable-local-auth false --name "S0" --tags \
+test="true" --account-name "myMapsAccount" --resource-group "myResourceGroup" --sku "S0"
+      - name: Create Gen2 Account
+        text: |-
+               az maps account create --kind "Gen2" --disable-local-auth true --name "G2" --tags \
+test="true" --account-name "myMapsAccount" --resource-group "myResourceGroup" --sku "S0"
```

Comment:
what's the difference between `--name` and `--account-name`?

Reply:
I think there must be some breaking change compare with released command arguments

Reply:
`--name` originally stands for sku, it will be changed to `--sku` later